src/test/java/teammates/test/cases/ui/AllActionsAccessControlTest.java

```diff
@@ -73,7 +73,7 @@ private static void addUnregStudentToCourse1() throws Exception {
     }
 
     @Test
-    public void InstructorCoursesPage() throws Exception {
+    public void instructorCoursesPage() throws Exception {
```

Comment:
I will temporarily change this to `test*` then. The other 1000+ occasions can be discussed in a separate issue, since the changes will definitely be large enough for a separate PR anyway.